dap/thread.go

```diff
@@ -110,8 +119,13 @@ func (t *thread) needsDebug(target digest.Digest, step stepType, err error) (rea
 	if err != nil {
 		reason = "exception"
 		desc = "Encountered an error during result evaluation"
-	} else if target != "" && step == stepNext {
+	} else if step == stepNext && target != "" {
 		reason = "step"
+	} else if step == stepContinue {
+		if _, ok := t.bps[target]; ok {
```

Comment:
Doesn't this need to be passed to the editor via `StoppedEvent.hitBreakpointIds`?

Reply:
Fixed.